web/pom.xml

```diff
@@ -875,10 +841,221 @@
+                      </fileMappers>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Copy schema resources using relative paths -->
+    <profile>
+      <id>schemas-copy</id>
+      <activation> <!-- use -DcopySchemas=true to activate       -->
+        <property><name>schemasCopy</name><value>true</value></property>
+      </activation>
+      <!-- Add dependencies to schema plugin having a custom Bean to be loaded. -->
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>schema-iso19115-3.2018</artifactId>
+          <version>${gn.schemas.version}</version>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>copy-schemas-ant</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                    <!-- Copy all the src/main/plugin folders in schemas to the webapp folder
+                    in order to be loaded by the application when running mvn jetty:run.
+                    -->
+                  <target description="copy from schemas to schema_plugins folder">
+                    <copy todir="${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins">
+                      <fileset dir="${basedir}/../schemas/">
+                        <include name="**/src/main/plugin/**"/>
+                      </fileset>
+                      <regexpmapper handledirsep="yes"
+                        from="^[-_a-zA-Z0-9]+/src/main/plugin/(.*)"
+                          to="\1" />
+                    </copy>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-resources-plugin</artifactId>
+            <!-- copy schema resources using relative paths -->
+            <executions>
+              <execution>
+                <id>copy-schemas</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <includeEmptyDirs>true</includeEmptyDirs>
+                  <overwrite>true</overwrite>
+                  <outputDirectory>${build.webapp.resources}</outputDirectory>
+                  <resources>
+                    <!-- Copy src/main/plugin folder to webapp folder source directory
```

Comment:
Shouldn't the copy resource for schema-iso19115-3.2018 be here?  And iso19139.xyz can be removed in this case as iso19139.xyz is not supplied as a sample in all the other locations.

Reply:
The execution `copy-schemas-ant` above handles all the contents in the `schemas` folder.

This example is provided if folks need to copy in something that is not in the `schemas` folder.

Reply:
Ian / Jose your changes to `add-schema.sh` work as expected (thank you).

Ian your change to` add-schema.sh` fill in this section, while it does no harm it is no longer required (and only useful if you are working with a schema plugin that is not in the schemas folder.